src/ddmd/hdrgen.d

```diff
@@ -1888,15 +1899,8 @@ public:
 
     override void visit(DtorDeclaration d)
     {
-        if (d.storage_class & STCtrusted)
-            buf.writestring("@trusted ");
-        if (d.storage_class & STCsafe)
-            buf.writestring("@safe ");
-        if (d.storage_class & STCnogc)
-            buf.writestring("@nogc ");
-        if (d.storage_class & STCdisable)
-            buf.writestring("@disable ");
-
+        if(stcToBuffer(buf, d.storage_class))
```

Comment:
If the hdrgen is run at the point in time where it should we should not have inferred attribs in here.